builders/ethereum/dev-env/tenderly.md

```diff
@@ -77,28 +75,31 @@ After a contract has been added, it will appear in the list of contracts on the
 
-### Create a Fork {: #fork-moonbeam }
+### Create a Virtual TestNets {: #virtual-testnets-moonbeam }
 
-Tenderly's forking feature simulates the live Moonbeam network in an isolated environment, which enables you to interact with deployed contracts and live on-chain data.
+Tenderly's Virtual TestNets feature simulates the live Moonbeam network in an isolated environment, which enables you to interact with deployed contracts and live on-chain data.
 
-There are some limitations to be aware of when using Tenderly's forking feature. You cannot interact with any of the [Moonbeam precompiled contracts](/builders/ethereum/precompiles/){target=\_blank} and their functions. Precompiles are a part of the Substrate implementation and therefore cannot be replicated in the simulated EVM environment. This prohibits you from interacting with cross-chain assets on Moonbeam and Substrate-based functionality such as staking and governance.
+There are some limitations to be aware of when using this feature. You cannot interact with any of the [Moonbeam precompiled contracts](/builders/ethereum/precompiles/){target=\_blank} and their functions. Precompiles are a part of the Substrate implementation and, therefore, cannot be replicated in the simulated EVM environment. This prohibits you from interacting with cross-chain assets on Moonbeam and Substrate-based functionality such as staking and governance.
 
-Tenderly makes creating a fork through the dashboard quite simple. To get started, click on **Forks** on the left-side menu and then click **Create Fork**. From there, you can take the following steps:
+Tenderly makes creating a TestNet through the dashboard quite simple. To get started, click on **Virtual TestNets** on the left-side menu and then click **Create Virtual TestNet**. From there, you can take the following steps:
 
-1. Select **Moonbeam** or **Moonriver** from the **Network** dropdown
-2. (Optional) Give your fork a name
-3. If you only need data up until a specific block, you can toggle the **Use Latest Block** slider to off and specify the block number. Otherwise, you can leave the slider as is to include all blocks up until the latest block
-4. Click **Create**
+1. Select **Moonbeam**, **Moonriver**, or **Moonbase Alpha** from the **Network** dropdown
+2. (Optional) Give your TestNet a name
+3. Select your **Chain ID**; you can use a custom one or the original network ID. It is recommended to set a custom Chain ID to prevent replay attacks and avoid issues when adding the Virtual TestNet to wallets
+4. Choose whether to turn on or off the **Public Explorer**
+5. Enable **State Sync** if you want to keep your Virtual TestNet updated in real-time with the parent network
+6. To limit data to a specific block, disable **Use Latest Block** and enter the block number. Otherwise, keep it enabled to include all blocks up to the latest
```

Comment:
```suggestion
6. To limit data to a specific block, disable **Use latest block** and enter the block number. Otherwise, keep it enabled to include all blocks up to the latest
```

To match the UI